src/pages/js/teamMembers.js

```diff
@@ -0,0 +1,93 @@
+import Suzy from '../../assets/Members/Suzy_MET.png';
+import Cody from '../../assets/Members/Cody_MET.png';
+import Arek from '../../assets/Members/Arek_MET.png';
+import Daniel from '../../assets/Members/Daniel_MET.png';
+import Fabrice from '../../assets/Members/Fabrice_MET.png';
+import Fardeen from '../../assets/Members/Fardeen_MET.png';
+import Pri from '../../assets/Members/Pri_MET.png';
+import Rya from '../../assets/Members/Rya_MET.png';
+import Sab from '../../assets/Members/Sab_MET.png';
+import Will from '../../assets/Members/Will_MET.png';
+import Zach from '../../assets/Members/Zach_MET.png';
+
+const teamMembers = [
+  {
+    name: 'Suzy Fendrick',
+    role: 'Designer',
+    location: 'Omaha, US',
+    image: Suzy,
+    tags: ['UX', 'UI', '+2']
```

Comment:
Wait, isn't this plus 2 supposed to be dynamic?

Reply:
Yup, was just doing this for now cause we need to discuss how we'd like to implement it